content/09.outlook.md

```diff
@@ -14,14 +14,14 @@ In particular, we have identified the following two areas as important for the f
 
 While there are many techniques to collate rare data from different sources, low-quality data may hurt the end goal even if it increases the size of the dataset.
 In our experience, collaboration with domain experts has proved to be critical in gaining insight into potential sources of variation in the datasets.
-An anecdotal example from the authors' personal experience: conversations with a rare disease clinician revealed that samples in a particular tumor dataset were collected using vastly different surgical techniques (laser ablation and excision vs standard excision). (TODO: This is duplicated in combining-datasets.md. should remove from one)
+An anecdotal example from the authors' personal experience: conversations with a rare disease clinician revealed that samples in a particular tumor dataset were collected using vastly different surgical techniques (laser ablation and excision vs standard excision).
 This information that was not readily available to non-experts, but was obvious the clinician. 
-Such instances underline the fact that continuous collaboration with domain experts and the sharing of well-annotated data is needed to generate robust datasets in the future.
+Such instances underline the fact that continuous collaboration with domain experts and the sharing of well-annotated data is needed to generate robust datasets in the future. [TODO: Is this sufficiently different from the combining datasets section now?]
```

Comment:
...but do we think it's sufficiently different now?

Reply:
imho.. yes :)